base_bank_branch/README.rst

```diff
@@ -0,0 +1,9 @@
+Bank Branches
```

Comment:
Hmm.. Isn't 'hierarchy' more suitable here?
```suggestion
Bank Hierarchy
```

Reply:
Hierarchy is too generic. It says nothing about branches, where branch is real term regarding banks structure.